### ISSUE TITLE
Put feature list on website/in repo

### ISSUE THREAD

**Issue by clarfonthey** [florence-social/mastodon-fork#66]
This would essentially just aggregate the change logs from each release into a nice, easy-to-access list which includes all of the differences between the fork and the mainline version.

It doesn't need to be terribly pretty at first, but it should have a static location and ideally contain pictures in the future. We can draw inspiration from glitch.social's page, but that hasn't been updated in a while.

**Comment by mal0ki** [comment]
Do we need this for the current milestone, or move it to the next one where we actually start merging more stuff?

**Comment by clarfonthey** [comment]
Yeah, we can push this onto pre-release 0.2 considering how 0.1 has next to nothing.
